selendroid-standalone/src/test/java/io/selendroid/standalone/android/impl/AbstractDeviceTest.java

```diff
@@ -49,10 +49,16 @@ public void testGetCrashLogContents() {
     when(device.getCrashLog()).thenCallRealMethod();
     when(
         device.executeCommandQuietly(argThat(matchesCmdLine("adb shell ls /storage/"))))  // The trailing '/' is key
-        .thenReturn("some_file\nappcrash.log\nanother_file");
+        .thenReturn(String.format("some_file%nappcrash.log%nanother_file"));
+    when(
+        device.executeCommandQuietly(argThat(matchesCmdLine("adb.exe shell ls C:\\storage/"))))  // The trailing '/' is key
```

Comment:
well then... you've encountered another actual bug in selendroid's product! :-D  (yay for tests?)

AbstractDevice.java needs to be adjusted to always return unix style paths instead of unix and will need to be fixed.